src/query-builder.ts

```diff
@@ -36,8 +37,8 @@ export function fql(
  * function
  */
 export class Query {
-  readonly #queryFragments: ReadonlyArray<string>;
-  readonly #queryArgs: (QueryValue | Query)[];
+  #queryFragments: ReadonlyArray<string>;
+  #queryArgs: (QueryValue | Query)[];
```

Comment:
These should still stay readonly.

```suggestion
  readonly #queryFragments: ReadonlyArray<string>;
  readonly #queryArgs: (QueryValue | Query)[];
```